clang/docs/BoundsSafetyAdoptionGuide.rst

```diff
@@ -0,0 +1,87 @@
+======================================
+Adoption Guide for ``-fbounds-safety``
+======================================
+
+.. contents::
+   :local:
+
+Where to get ``-fbounds-safety``
+================================
+
+The open sourcing to llvm.org's ``llvm-project`` is still on going and the
+feature is not available yet. In the mean time, the preview implementation is
+available
+`here <https://github.com/swiftlang/llvm-project/tree/stable/20240723>`_ in a
+fork of ``llvm-project``. Please follow
+`Building LLVM with CMake <https://llvm.org/docs/CMake.html>`_ to build the
+compiler.
+
+Feature flag
+============
+
+Pass ``-fbounds-safety`` as a Clang compilation flag for the C file that you
```

Comment:
@rapidsna I thought `-fbound-safety` was broken in open source? I thought people needed to do `-Xclang -fbounds-safety`.. or something like that?

Reply:
No. `-fbounds-safety` is available as a driver in the preview implementation I pointed out.

Reply:
Note that the purpose of this document is to instruct people on how to use the preview implementation.